src/main/java/org/gridsuite/study/server/service/shortcircuit/ShortCircuitService.java

```diff
@@ -291,4 +329,39 @@ public void assertShortCircuitAnalysisNotRunning(UUID nodeUuid) {
             throw new StudyException(SHORT_CIRCUIT_ANALYSIS_RUNNING);
         }
     }
+
+    public static List<VoltageRange> getVoltageRanges(boolean isNominal) {
+        List<VoltageRange> voltageRanges = new ArrayList<>();
+        voltageRanges.add(new VoltageRange(20.0, isNominal ? 20.0 : 22.0, isNominal ? 1 : 1.1));
+        voltageRanges.add(new VoltageRange(45.0, isNominal ? 45.0 : 49.5, isNominal ? 1 : 1.1));
+        voltageRanges.add(new VoltageRange(63.0, isNominal ? 63.0 : 69.3, isNominal ? 1 : 1.1));
+        voltageRanges.add(new VoltageRange(90.0, isNominal ? 90.0 : 99.0, isNominal ? 1 : 1.1));
+        voltageRanges.add(new VoltageRange(150.0, isNominal ? 150.0 : 165.0, isNominal ? 1 : 1.1));
+        voltageRanges.add(new VoltageRange(225.0, isNominal ? 225.0 : 245.0, isNominal ? 1 : 1.09));
+        voltageRanges.add(new VoltageRange(380.0, isNominal ? 400 : 420.0, isNominal ? 1 : 1.09));
+        return voltageRanges;
+    }
+
+    public static ShortCircuitParametersInfo toShortCircuitParametersInfo(ShortCircuitParametersEntity entity) {
+        Objects.requireNonNull(entity);
+        Map<String, List<VoltageRange>> voltageRangesMap = new HashMap<>();
+        voltageRangesMap.put(InitialVoltageProfileMode.NOMINAL.toString(), getVoltageRanges(true));
+        voltageRangesMap.put(InitialVoltageProfileMode.CONFIGURED.toString(), getVoltageRanges(false));
```

Comment:
You can make it more simple. One constant to define nominal voltage mal, another one to define CEI909 voltage map. And not using a function

Reply:
I don't completely agree. using 2 constants causes code duplication unless I misunderstood your suggestion.
In addiction to that getVoltageRanges is used also when we runShortCircuit and duplicate study( insertDuplicatedStudy)

Reply:
It's really small code duplication and it makes the code clearer. It shows clearly that these are constant configurations.
In the future, we'll have other configurations, we won't be able to distinguish them wth a boolean "isNominal"

Reply:
And the key for this entry map is more CEI909 than CONFIGURED

Reply:
Fixed.